### ISSUE TITLE
[1.9.4] iPhone 6 / iOS 9.3.3 - Twitter Login Failed - Open linked account and click on Twitter icon

### ISSUE THREAD

**Issue by hiteshbheda** [Lomotif/lomotif-ios-issues#530]
Frequency: Every time

**_Pre-requisite: User should be logged out (Unlink) from Twitter account_**

Action:
1. Launch Lomotif
2. Click on Menu button from Top Right corner of thescreen
3. Tap on Linked Accounts icon
4. Click on Twitter option
5. Check whether you are redirected to login page and allow you to process or not

Expected result: User should be redirected to Login screen and allow user to link Twitter account

Actual result: 
1. It does not redirect user to Login screen and shows message "Login Failed" 
2. Sometime if it redirect user to login screen then after sometime it display "Login Failed" message

![img_3580](https://cloud.githubusercontent.com/assets/9691390/17169349/a524ca80-5404-11e6-9ecc-915661f3cda7.PNG)

![img_3581](https://cloud.githubusercontent.com/assets/9691390/17169350/a532d026-5404-11e6-9884-8677df83504f.PNG)

**Comment by xiuie** [comment]
I can't reproduce this on my iPhone 6 / iOS 10.

**Comment by vysohanych** [comment]
@xiuie I can't reproduce it on build 1.9.4 on iPhone 5S iOS 8.4.1

**Comment by hiteshbheda** [comment]
@xiuie I can't reproduce this Error message now for build [1.9.4] on iPhone 6 / iOS 9.3.3

**Comment by testershub** [comment]
I can reproduced this issue for build [1.9.0 (1.9.4)] on iPhone 6S /9.3.2, iPad Mini 3/9.3.2 by following steps-
Action:
1. Launch Lomotif.
2. Click on Menu button from Top Right corner of the screen.
3. Turn OFF internet connection.
4. Tap on Linked Accounts icon.
5. Click on "Twitter" option.
6. Check if user is navigated to "Menu options" screen and error message as "Login Failed" is displayed.

Actual Result: Error message as "Login Failed" is displayed when user taps on "Twitter" option on "Linked Accounts" screen whereas internet connection is disabled in background.

**Comment by xiuie** [comment]
@testershub Thanks so much for the clarification about the internet connection

**Comment by hiteshbheda** [comment]
@xiuie I agree with @testershub When internet if Turned off this issue is reproducible otherwise not.

Thanks @testershub
